### PR TITLE
fix(frontend): fixed term planner to show empty grid box when no course selected. 

### PR DESCRIPTION
**Pull request by pikapiku**
- instead of a loading status on empty course select on term planner, it now prints an empty grid

### CODE REVIEW

File: frontend/src/pages/TermPlanner/TermPlanner.tsx [hunk 1]
```diff
@@ -272,7 +272,7 @@ const TermPlanner = () => {
           // TODO: Fix Suspense by updating to react-query v5
           /* <Suspense fallback={<Spinner text="Loading Table..." />}> */
         }
-        {courseQueries.some((c) => c.isPending) || !courseInfos[LIVE_YEAR] ? (
+        {courseQueries.some((c) => c.isPending || coursesQuery.isPending || plannerQuery.isPending )  ? (
```
Comment (lhvy) [inline frontend/src/pages/TermPlanner/TermPlanner.tsx]: What motivated this change? Also, I don't think the added content should be inside the call to `some`.

Comment (pikapiku) [reply frontend/src/pages/TermPlanner/TermPlanner.tsx]: good spotting. i'll fix this and send a pr again

**Review by ollibowers — CHANGES_REQUESTED** [review]
looks good to me so far, if you could, please rebase off 331-staging with lucas changes, and then test out a course that has no legacy offering, (such as COMP6131 with a start year of 2021)

**Comment by ollibowers** [comment]
> looks good to me so far, if you could, please rebase off 331-staging with lucas changes, and then test out a course that has no legacy offering, (such as COMP6131 with a start year of 2021)

331-staging doesnt work with this even, so blocking PR until it gets fixed in 331-staging

**Comment by ollibowers** [comment]
this has been merged inn #1160

**Pull request closed without merging**